packages/jingoo/jingoo.1.2.8/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+name: "jingoo"
+version: "1.2.8"
```

Comment:
can you also remove the `name` and `verision` fields from the opam file? they already appear in the directory name, no need to repeat them (and that will remove possible source of confusion later). Thanks!